### DISCUSSION TITLE
How do I make my own unblocked game website on iPad

### DISCUSSION THREAD

**Discussion by coolerfaster** [UseInterstellar/Interstellar#334] [General]
Also if it is on computer how would I do that can you give me some directions?

**Comment by 3khoisthebest** [comment]
https://docs.google.com/document/d/1zPRe9uhQMYx5G5pGtrk4wcqlvooPDpk6k0i0W9hJ5kI/edit?usp=sharing
i made it a few days ago, if it's too complicated, watch the vid on my yt
channel (it might not show up on a school device, since it's restricted)
***@***.***
